### PR TITLE
Added support for Fedora in install-gui script

### PR DESCRIPTION
**Pull request by feldsam**

### CODE REVIEW

File: install-gui.sh [hunk 1]
```diff
@@ -23,17 +23,17 @@ if [ "$(uname)" = "Linux" ]; then
...
-	elif type yum && [ ! -f /etc/rocky-release ] && [ -f /etc/redhat-release ] || [ -f /etc/centos-release ]; then
+	elif type yum && [ ! -f /etc/rocky-release ] && [ ! -f /etc/fedora-release ] && [ -f /etc/redhat-release ] || [ -f /etc/centos-release ]; then
 		# CentOS or Redhat
 		echo "Installing on CentOS/Redhat."
 		curl -sL https://rpm.nodesource.com/setup_12.x | sudo bash -
 		sudo yum install -y nodejs
-	elif type yum && [ -f /etc/rocky-release ]; then
+	elif type yum && [ -f /etc/rocky-release ] || [ -f /etc/fedora-release ]; then
                 # RockyLinux
                 echo "Installing on RockyLinux"
```
Comment (arvidn) [inline install-gui.sh:38]: for consistency, perhaps this printout and comment should be updated to include Fedora too

**Review by arvidn — DISMISSED** [review]
looks reasonable to me

**Review by hoffmang9 — APPROVED** [review]
aok

**Comment by hoffmang9** [comment]
Thank you!

**Pull request merged**